### PR TITLE
ROSHAN ISSUE #1919

### PR DESCRIPTION
**Pull request by Roshanbhadauriya**
Broken Links Audit: Entries 41-50. Solved

### Description

Fixed all broken links on files:

- docs/content/uiux/concepts/sketching/sketching.md
- docs/content/uiux/concepts/homepage/homepage.md
- docs/content/uiux/concepts/design-software/design-software.md
- docs/content/uiux/concepts/wireframe/wireframe.md
- docs/content/uiux/concepts/search-fields/search-fields.md
- docs/content/uiux/concepts/image/image.md
- docs/content/uiux/concepts/wireframe-elements/wireframe-elements.md
- docs/content/uiux/concepts/button/button.md
- docs/content/uiux/concepts/opacity/opacity.md
- docs/content/uiux/concepts/qualitative-research/qualitative-research.md

### Type of Change

Editing an existing entry (fixing a typo, bug, issues, etc)

- Editing an existing entry (fixing a typo, bug, issues, etc)

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/uiux/concepts/wireframe/wireframe.md [hunk 1]
```diff
@@ -13,8 +13,8 @@ CatalogContent:
...
+Wireframes are like rough drafts that help determine a solution for the structure ([information architecture](https://www.codecademy.com/resources/docs/uiux/information-architecture) and [interaction design](https://www.codecademy.com/resources/docs/uiux/ux-design)) and skeleton ([interface design](https://www.codecademy.com/resources/docs/uiux/five-elements-of-ux-design), [navigation design](https://www.codecademy.com/resources/docs/uiux/five-elements-of-ux-design), [information design](https://www.codecademy.com/resources/docs/uiux/five-elements-of-ux-design)) layers of the [Five Elements of UX Design](https://www.codecademy.com/resources/docs/uiux/five-elements-of-ux-design).
```
Comment (caupolicandiaz) [inline content/uiux/concepts/wireframe/wireframe.md:20]:
```suggestion
Wireframes are like rough drafts that help determine a solution for the structure ([information architecture](https://www.codecademy.com/resources/docs/uiux/information-architecture) and [interaction design](https://www.codecademy.com/resources/docs/uiux/ux-design)) and skeleton ([interface design](https://www.codecademy.com/resources/docs/uiux/ui-design), [navigation design](https://www.codecademy.com/resources/docs/uiux/navigation), information design) layers of the [Five Elements of UX Design](https://www.codecademy.com/resources/docs/uiux/five-elements-of-ux-design).
```

Comment (caupolicandiaz) [reply content/uiux/concepts/wireframe/wireframe.md:20]: we don't have a current resource for information-design, so I removed the link

File: content/uiux/concepts/design-software/design-software.md [hunk 2]
```diff
@@ -11,6 +11,6 @@ CatalogContent:
   - 'paths/front-end-engineer-career-path'
 ---
 
-**Design software** is useful when creating user interface (UI) and user experience (UX) deliverables such as [wireframes](https://www.codecademy.com/resources/docs/uiux/wireframe), interactive prototypes, [personas](https://www.codecademy.com/resources/docs/uiux/persona), [user journey maps](https://www.codecademy.com/resources/docs/uiux/user-journey-map), and [site maps](https://www.codecademy.com/resources/docs/uiux/site-map), to name a few.
+**Design software** is useful when creating user interface (UI) and user experience (UX) deliverables such as [wireframes](https://www.codecademy.com/resources/docs/uiux/wireframe), interactive prototypes, [personas](https://www.codecademy.com/resources/docs/uiux/persona), [user journey maps](https://www.codecademy.com/resources/docs/uiux/user-workflows), and [site maps](https://www.codecademy.com/resources/docs/uiux/site-map), to name a few.
```
Comment (caupolicandiaz) [inline content/uiux/concepts/design-software/design-software.md:14]:
```suggestion
**Design software** is useful when creating user interface (UI) and user experience (UX) deliverables such as [wireframes](https://www.codecademy.com/resources/docs/uiux/wireframe), interactive prototypes, [personas](https://www.codecademy.com/resources/docs/uiux/persona), [user workflows](https://www.codecademy.com/resources/docs/uiux/user-workflows), and [site maps](https://www.codecademy.com/resources/docs/uiux/site-map), to name a few.
```

**Review by caupolicandiaz — CHANGES_REQUESTED** [review]
@Roshanbhadauriya I've completed my review, there are just a few edits to be made. Please update the entry, and tag me when you've finished making changes.

**Comment by Roshanbhadauriya** [comment]
@caupolicandiaz I have updated the changes that you suggest.

**Review by caupolicandiaz — APPROVED** [review]
@Roshanbhadauriya Looks good.

**Comment by Roshanbhadauriya** [comment]
@caupolicandiaz Thank you for the opportunity, this is my contribution.

File: content/uiux/concepts/qualitative-research/qualitative-research.md [hunk 3]
```diff
@@ -26,4 +26,4 @@ Qualitative [attitudinal](https://www.codecademy.com/resources/docs/uiux/attitud
 - [Adobe XD: The “Why” Behind Qualitative User Research](https://xd.adobe.com/ideas/process/user-research/why-behind-qualitative-user-research/)
 - [Nielsen Norman Group: How to Analyze Qualitative Data from UX Research: Thematic Analysis](https://www.nngroup.com/articles/thematic-analysis/)
 - [CareerFoundry: A Beginner’s Guide to Qualitative UX Research](https://careerfoundry.com/en/blog/ux-design/qualitative-ux-research/)
-- [UserZoom: UX research methods - a comprehensive guide](https://www.userzoom.com/ux-library/ux-research-methods-a-comprehensive-guide/)
+- [UserZoom: UX research methods - a comprehensive guide](https://www.userzoom.com/ux-research/)
```
Comment (yangc95) [inline content/uiux/concepts/qualitative-research/qualitative-research.md:29]:
```suggestion
- [UserZoom: UX research methods - a comprehensive guide](https://www.userzoom.com/ux-research/methods/)
```

**Review by yangc95 — APPROVED** [review]

**Comment by yangc95** [comment]
Thanks @Roshanbhadauriya for your contribution!

Edits should appear here: UIUX: [Button](https://www.codecademy.com/resources/docs/uiux/button), [Design Software](https://www.codecademy.com/resources/docs/uiux/design-software), [Homepage](https://www.codecademy.com/resources/docs/uiux/homepage), [Image](https://www.codecademy.com/resources/docs/uiux/image), [Opacity](https://www.codecademy.com/resources/docs/uiux/opacity), [Qualitative Research](https://www.codecademy.com/resources/docs/uiux/qualitative-research), [Search Fields](https://www.codecademy.com/resources/docs/uiux/search-fields), [Sketching](https://www.codecademy.com/resources/docs/uiux/sketching), [Wireframe Elements](https://www.codecademy.com/resources/docs/uiux/wireframe-elements), [Wireframe](https://www.codecademy.com/resources/docs/uiux/wireframe)

@caupolicandiaz

**Pull request merged**